src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private int getRandomNumber() {
+        return new Random().nextInt(100);
+    }
+
+    private String getRundomColor() {
+        return new ColorSupplier().getRandomColor();
+    }
```

Comment:
Lets create to fields, where you declare and initialize Random and ColorSupplier objects, and then use it inside getRandomBall method